compiler/backend/presLangScript.sml

```diff
@@ -16,8 +16,14 @@ val _ = new_theory"presLang";
 (* Special operator wrapper for presLang *)
 val _ = Datatype`
   op =
-    | Ast ast$op
-    | Conlang conLang$op`;
+    | Ast_op ast$op
+    | Con_op conLang$op`;
```

Comment:
I mostly felt it would be too long to write out the "lang" part every time, since it might get written a lot. But since it is only in presLang, and it probably won't bother the main development, adding "lang" seems sensible to me.